src/ui/components/Collections/Collections.jsx

```diff
@@ -1,81 +1,215 @@
-import React from 'react';
+import React, { useState } from 'react';
 import axios from 'axios';
 import { useQuery } from 'react-query';
 import {
   Table,
   TableBody,
   TableCell,
   TableContainer,
-  TableHead,
-  TableRow
+  TableRow,
+  TablePagination
 } from '@material-ui/core';
 import ReactJson from 'react-json-view';
 import PropTypes from 'prop-types';
+import useStyles from './styles';
+import SortedTableHead from './SortedTableHead';
+import DeleteIcon from '@material-ui/icons/Delete';
+import CreateIcon from '@material-ui/icons/Create';
+import Button from '@material-ui/core/Button';
+import AddIcon from '@material-ui/icons/Add';
 
 const Collections = props => {
+  const classes = useStyles();
   const { selectedCollection } = props;
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+  const [order, setOrder] = useState('asc');
+  const [orderBy, setOrderBy] = useState('');
+
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  function descendingComparator(a, b, orderBy) {
+    if (b[orderBy] < a[orderBy]) {
+      return -1;
+    }
+    if (b[orderBy] > a[orderBy]) {
+      return 1;
+    }
+    return 0;
+  }
+
+  function serverStatus(server) {
+    return server && true;
+  }
+
+  function getComparator(order, orderBy) {
+    return order === 'desc'
+      ? (a, b) => descendingComparator(a, b, orderBy.toLowerCase())
+      : (a, b) => -descendingComparator(a, b, orderBy.toLowerCase());
+  }
+
+  function stableSort(array, comparator) {
+    const stabilizedThis = array.map((el, index) => [el, index]);
+
+    stabilizedThis.sort((a, b) => {
+      const order = comparator(a[0], b[0]);
+      if (order !== 0) return order;
+      return a[1] - b[1];
+    });
+    return stabilizedThis.map(el => el[0]);
+  }
+
+  const handleRequestSort = (event, property) => {
+    const isAsc = orderBy === property && order === 'asc';
+    setOrder(isAsc ? 'desc' : 'asc');
+    setOrderBy(property);
+  };
+
+  const handleChangeRowsPerPage = event => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
 
   const { data } = useQuery(['collections', { selectedCollection }], () =>
     axios.get(`http://localhost:3000/collection/${selectedCollection}`)
```

Comment:
I realize this is not your change but I'm just seeing it now. This should not be hardcoded to localhost:3000. Will this work?
```suggestion
    axios.get(`/collection/${selectedCollection}`)
```